<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>class_tab</class>
 <widget class="QWidget" name="class_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="BT_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel changes</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_apply">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Apply changes</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/tick.png</normaloff>:/icons/tick.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QSpinBox" name="F_class_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="F_class_description">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="F_class_name"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="CB_view"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="BT_add_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_sub_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="fields_list">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Field</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mandatory</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Default</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="placeholderText">
      <string>test</string>
     </property>
     <item>
      <property name="text">
       <string>dfgdytk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sgfhsfghs</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="BT_add_ref">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BT_sub_ref">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="ref_list">
     <property name="indentation">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>Reference</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Multiple</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mandatory</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Classes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
